### PR TITLE
WIP:MCH & OPD feedback

### PR DESCRIPTION
**Pull request by moshonk**
@ssmusoke Please find this for your review.
I worked on the following:
**OPD** 
• For all forms where HBP is captured, systolic should preceed diastolic. 
• As much as possible, units of measurement should be indicated
	? Temp Temperature (oC)
	? Blood Sugar è Blood Sugar (mg/dl)
• New diagnosis – once a diagnosis is selected, it should disappear from list
• Drug - once a drug is selected, it should disappear from list

**ANC Register -** 
• Make ANC Number not required *
• units of measurement should be indicated
	? Temp Temperature (oC)
	? Blood Sugar è Blood Sugar (mg/dl)
• For all forms where HBP is captured, systolic should proceed diastolic

**Maternity Register** 
• Make Date of Delivery not required as there are users who enter data continously 
• Enable Condition of Baby needs to be multiple select as a baby can have multiple conditions 
• Management of 3rd stage labour should enable multiple selection as multiple management options may be selected 
• Add TR+ to the EMTCT codes for both woman and partner
• Condition of mother on discharge should enable the multiple selection
cc @ningosi

### CODE REVIEW

**Comment by ssmusoke** [comment]
@moshonk Maternity form is broken with java.lang.IllegalArgumentException: Cannot find concept for value dc66a5b3-30ab-102d-86b0-7a5022ba4115 in answerConceptId attribute value. Parameters: {conceptId=1284AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA, style=checkbox, answerLabel=Puerperal sepsis, answerConceptId=dc66a5b3-30ab-102d-86b0-7a5022ba4115}

**Comment by moshonk** [comment]
@ssmusoke Please check, you may have restored the wrong initial backup file. That concept is available in the **concept_dictonary_ref_1.0.13.sql** backup file. 
This PR doe not not make any changes to that section of the form neither does it affect it i the in Data Exchange files

File: omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml [hunk 1]
```diff
@@ -757,24 +773,25 @@
...
+              <obs conceptId="f9573837-6a9b-4f2f-9298-aecf27b15fb3" allowTime="true" class="date-and-time" required="false"/>
             </p>
           </td>
-          <td>
+          <td colspan="2">
             <p>
               <label>Management of 3rd stage labour</label>
-              <obs conceptId="425458af-d1f7-40e4-a672-7d9e38aedb3c" answerConceptIds="e123d685-812a-43c3-bc05-db4e14d8c05c,eca9da28-31d3-4e6f-828d-441e9237b7a5,1c4323a3-6cc6-44d0-81ee-839014bca19c"
-                   answerLabels="1.Ergometrine,2.Oxtocin (Pitocin),3.Misoprostol" />
+              <obs conceptId="425458af-d1f7-40e4-a672-7d9e38aedb3c" answerConceptId="e123d685-812a-43c3-bc05-db4e14d8c05c" answerLabel="1.Ergometrine" />
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml]: Answer label 1 - Ergometrine

File: omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml [hunk 2]
```diff
@@ -757,24 +773,25 @@
...
+              <obs conceptId="425458af-d1f7-40e4-a672-7d9e38aedb3c" answerConceptId="eca9da28-31d3-4e6f-828d-441e9237b7a5" answerLabel="2.Oxtocin (Pitocin)" />
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml]: Answer label: 2 - Oxtocin (Pitocin)

File: omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml [hunk 3]
```diff
@@ -757,24 +773,25 @@
...
+              <obs conceptId="425458af-d1f7-40e4-a672-7d9e38aedb3c" answerConceptId="1c4323a3-6cc6-44d0-81ee-839014bca19c" answerLabel="3.Misoprostol" />
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml]: Answer label: 3 - Misoprostol

**Comment by ssmusoke** [comment]
@moshonk 

1. ANC number has to be a concept not an identifier, and its still required

2. I am still unable to select options in the different columns of the first row of Final Diagnosis 

3. Condition of baby - put AL on a new line 

4. Condition of mother at discharge should remain a drop down 

5. Management of 3rd Stage labor - put each option on its own line since the space exists

6. Routine Medication - Other Specify make it cover 2 columns as the space is there 

7. Discharge Information - Family Planning method - give it a row above the options and make it bold so that we can use the space. Leave the 3 column arrangement for the values

**Comment by moshonk** [comment]
@ssmusoke, I have acted upon the requested changes. Please check them out. Sorry for delay in responding, I've just seen the comments a few moments ago.

**Comment by moshonk** [comment]
@ssmusoke I have an update for your review:
Bug Fix: Final diagnosis Section - Selection of diagnosis on one column deselects a selection on another column

**Comment by moshonk** [comment]
@ssmusoke, the updates are ready

**Comment by ssmusoke** [comment]
@moshonk For an HIV+ woman on the maternity form I can still select HIV- and the related EMTCT codes for negative mothers - do check the Javascript on this page

**Pull request merged**